### ISSUE TITLE
Ask about zaibuyidao_Humanize Velocity

### ISSUE THREAD

**Issue by ARkaine2** [zaibuyidao/ReaScripts#3]
Can you please make (or tell how to make) this script work without window?
I need humanizing velocity per click (with definible amount of humanizing in .lua file).

And, if possible, can you make randomize velocity script with definible (in .lua file) minimum and maximum randomizible velocity (like in "zaibuyidao_Random Velocity (Limit Range)", but without window like "zaibuyidao_Random Velocity").

In other words:
I need 2 scripts:
Humanize Velocity
Randomize Velocity
That will:
Work without window.
Have definible (in .lua file) settings.

Thanks.

**Comment by zaibuyidao** [comment]
Ok, I get it. This is the window-removed version, please search the following script in ReaPack:
zaibuyidao_Humanize Velocity (Customize).lua
zaibuyidao_Random Velocity (Customize).lua

**Comment by ARkaine2** [comment]
Thank you a lot.
It works perfectly.
